docs/contributors/javascript-testing.md

```diff
@@ -0,0 +1,20 @@
+# JavaScript Testing
```

Comment:
At some point, it would be great to add a PHP Testing file too. (Not blocking this PR, tho).

Reply:
Agreed :)